src/DependencyManagement/DependencyManager.cs

```diff
@@ -325,8 +328,10 @@ private string GetManagedDependenciesPath(string functionAppRoot)
             }
             else
             {
-                // Otherwise, the ManagedDependencies folder is created under the function app root.
-                managedDependenciesFolderPath = Path.Join(functionAppRoot, ManagedDependenciesFolderName);
+                // Otherwise, the ManagedDependencies folder is created under LocalApplicationData\AzureFunctions\FunctionAppName\ManagedDependencies.
+                string functionAppName = new DirectoryInfo(functionAppRootPath).Name;
```

Comment:
nit: Use `Path.GetFileName` to not allocate a new `DirectoryInfo` object.

Reply:
Fixed.